lib/httpx/0.0.1/index.js

```diff
@@ -0,0 +1,178 @@
+import http from 'k6/http';
+
+class Request{
+  constructor(method, url, params){
+    params = params || {};
+    this.method = method;
+    this.url=url;
+    this.body = params['body'] || null;
+    delete params['body'];
+    this.params = params||{};
+  }
+
+  addHeader(name, value){
+    this.params.headers = Object.assign({}, this.params.headers, {[name]: value});
+    return this;
+  }
+
+  addTag(name, value){
+    this.params.tags = Object.assign({}, this.params.tags, {[name]: value});
+    return this;
+  }
+}
+
+class Get extends Request{
+  constructor(url, params){
+    super('GET', url, params)
+  }
+}
+
+class Post extends Request{
+  constructor(url, params){
+    super('POST', url, params)
+  }
+}
+
+class Put extends Request{
+  constructor(url, params){
+    super('PUT', url, params)
+  }
+}
+
+class Patch extends Request{
+  constructor(url, params){
+    super('PATCH', url, params)
+  }
+}
+
+class Options extends Request{
+  constructor(url, params){
+    super('OPTIONS', url, params)
+  }
+}
+
+class Delete extends Request{
+  constructor(url, params){
+    super('DELETE', url, params)
+  }
+}
+
+class Httpx{
+
+  constructor(){
```

Comment:
Would be nice to be able to pass the params and baseURL directly when creating a new session.
`new Httpx({ headers, tags, baseURL })`